jha/src/data-types/supp-ben-data.js

```diff
@@ -0,0 +1,253 @@
+import {
+  calculateAge,
+} from 'common-lib-vue';
+
+export default class SuppBenData{
+  #selectedNOAYear;
+  #ahIncome;
+  #ahAge;
+  #hasSpouse;
+  #spouseIncome;
+  #spouseAge;
+  #children;
+  #claimedChildCareExpenses;
+  #hasDisabilityCredit;
+  #selectedDisabilityRecipients;
+  #numDisabilityChildren;
+  #hasRDSP;
+  #rdspAmount;
+  #hasAttendantNursingExpenses;
+  #selectedAttendantNursingRecipients;
+  #numAttendantNursingChildren;
+  #attendantNursingReceipts;
+  #qualificationThreshhold;
```

Comment:
What's the purpose of the "#" prefix? I've never seen that before...

Reply:
While JS doesn't have any provision for having private instance variables(you can't stop someone accessing directly without using getters or setters) it is a standard convention that private variables are prefixed with a '#'.  Many linters will even throw out errors if someone tries to access one from outside the class.

Reply:
Interesting. Appears to be a new JS feature. I'm fine with this, but please make sure it still works in IE11. Babel will likely transpile this, but just to be sure.

Reply:
I have tested in IE and it seems to work fine... as this is only a naming convention it shouldn't make a difference to behavior even if babel doesn't transpile it.